src/connectors/components/WalletListModalConnector2.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import { connect } from 'react-redux'
+
+import type { StateProps } from '../../components/modals/WalletListModal2.js'
+import { WalletListModal } from '../../components/modals/WalletListModal2.js'
+import { getActiveWalletIds } from '../../modules/UI/selectors.js'
+import type { State } from '../../types/reduxTypes.js'
+
+export const mapStateToProps = (state: State): StateProps => {
```

Comment:
Please move the whole connector into the `modals/WalletListModal2.js` file. I am trying to get rid of the separate connector files, since they make the program really hard to read with all the flipping back & forth.